api/fwapi.js

```diff
@@ -0,0 +1,245 @@
+const axios = require('axios')

```

Comment:
I assume this is ported over from the original fwapi, looks good

Reply:
This file is a wrapper for using FWAPI yeah